NEWS.md

```diff
@@ -1,5 +1,11 @@
 # News
 
+## 2023-10
+
+### Changes in pre-built images
+
+- `rocker/binder` now includes necessary LaTeX packages to allow Jupyter Notebooks to be converted to PDF. ([#714](https://github.com/rocker-org/rocker-versioned2/pull/714))
+
```

Comment:
This is also a script change, so please write about that too.
(Since it is not possible to revert to the original behavior, I think it is necessary to emphasize the increased execution time and the size increasing of the image)

Reply:
@eitsupi how about now?

Reply:
Thanks!